forms-flow-documents/Dockerfile

```diff
@@ -0,0 +1,46 @@
+#Author: Kurian Benoy
+FROM python:3.9-slim-buster
+
+WORKDIR /forms-flow-api/app
```

Comment:
is this correct ?

Reply:
Changed name to match with the root folder name